src/main/steps/birth-father/address-known/content.ts

```diff
@@ -1,22 +1,81 @@
 import { YesOrNo } from '../../../app/case/definition';
-import { Radios } from '../../common/components/radios';
-import { SECTION, SECTION_IN_WELSH } from '../constants';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
 
-const enContent = {
-  section: SECTION,
+const en = () => ({
+  section: "Birth father's details",
```

Comment:
why this has changed?

Reply:
To sync with other page (content.ts)  and to avoid constant import in test case